soh/src/code/z_player_lib.c

```diff
@@ -1391,6 +1391,9 @@ Vec3f D_801261E0[] = {
     { 200.0f, 200.0f, 0.0f },
 };
 
+// OTRTODO: Figure out why this value works/what this value should be
```

Comment:
could you add a little comment describing how you landed at this value (started at max and worked backwards until things behaved properly)?

it'd be nice to provide a bit more context for anyone working in this area in the future